recipes/fish2eod/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "fish2eod" %}
+{% set version = "1.0a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/aaronshifman/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5ff1a9d5e5e8f10871ced326d4b86f9ecd0616e760e6beb983dfbebcae26f6c2
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - {{ pin_compatible('numpy') }}
+    - pip
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - fenics-dolfin ==2019.1.0
+    - superlu_dist ==7.2.0
+    - gmsh ==3.0.6
+    - pytables
+    - h5py
+    - matplotlib-base
+    - meshio <5
+    - {{ pin_compatible('numpy') }}
```

Since you're not linking against the numpy C-API, just do:
```suggestion
    - numpy
```